backend/census_historical_migration/sac_general_lib/sac_creator.py

```diff
@@ -81,6 +81,14 @@ def setup_sac(user, audit_header):
     sac.auditee_certification = auditee_certification(audit_header)
     sac.auditor_certification = auditor_certification(audit_header)
     sac.data_source = settings.CENSUS_DATA_SOURCE
+
+    if audit_header.ENTITY_TYPE == "Tribal":
+        sac.tribal_data_consent = {
+            "tribal_authorization_certifying_official_title": "GSA_MIGRATION",
+            "is_tribal_information_authorized_to_be_public": audit_header.SUPPRESSION_CODE != "IT",

```

Comment:
```suggestion
            "is_tribal_information_authorized_to_be_public": string_to_string(audit_header.SUPPRESSION_CODE).upper() != "IT",

```

Reply:
Are we supposed to make it public when `IT` or when not `IT` ? We may want to document this in the ticket.

Reply:
My understanding from the meeting with Carley is that if IT is not there, it's public. We can def verify before this goes out. Added that note to the ticket too.